### DISCUSSION TITLE
Primary key value while making a record

### DISCUSSION THREAD

**Discussion by thematrix011** [aminalaee/sqladmin#404] [Q&A] [answered]
I have a model User wherein the User.id is the primary field. While making a new entry into the db using the /admin route it doesn't give me an option to enter the id. Due to this any other entry I make will be duplicate.
One way is I enable auto increment for my primary field but what if I want varying values(of the correct type) for the primary key.
What should I do so it takes the primary field value when entering the details of a new entry of the model.

**Comment by aminalaee** [comment, marked as answer]
So if I understand correctly you want to enter the PK value in the create page, right?
If that is the case, I think you can use form_include_pk:
https://aminalaee.dev/sqladmin/configurations/
